sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/EventModel.kt

```diff
@@ -89,7 +88,8 @@ class EventModel(val sessionId: String) : BaseQueryModelView<Session, SessionInf
             }
 
             val params = HashMap<String, Any>()
-            params["slot"] = analyticsDateFormat.format(session.startsAt)
+            val analyticsDateFormat = DateFormatHelper("MM_dd_HH_mm")
```

Comment:
Moved these variables to inside the functions because they're getting the time zone from the ServiceRegistry and it hasn't been initialized yet.